cloudflare-mixin/config.libsonnet

```diff
@@ -1,16 +1,33 @@
 {
-  _config+:: {
-    dashboardTags: ['cloudflare-mixin'],
-    dashboardPeriod: 'now-30m',
-    dashboardTimezone: 'default',
-    dashboardRefresh: '1m',
+  local this = self,
+  enableMultiCluster: false,
+  filteringSelector: 'job="integrations/cloudflare"',
+  groupLabels: ['job', 'cluster', 'zone', 'script_name'],
+  instanceLabels: ['instance'],
 
-    // CloudflareMetricsDown alert filter variable
-    alertsMetricsDownJobName: 'integrations/cloudflare',
+  dashboardTags: [self.uid],
+  legendLabels: ['instance'],
+  uid: 'cloudflare',
+  dashboardNamePrefix: 'Cloudflare',
 
-    // alerts thresholds
-    alertsHighThreatCount: 3,  // count
-    alertsHighRequestRate: 150,  // percentage
-    alertsHighHTTPErrorCodeCount: 100,  // count
+  // additional params
+  dashboardPeriod: 'now-30m',
+  dashboardTimezone: 'default',
+  dashboardRefresh: '1m',
+  metricsSource: 'prometheus',
+
+  // CloudflareMetricsDown alert filter variable
+  alertsMetricsDownJobName: 'integrations/cloudflare',
```

Comment:
Minor nit, we should probably extract this and reuse it in the `filteringSelector` so that both are updated dynamically in case end users change it